tracer/src/Datadog.Trace/AppSec/SecuritySettings.cs

```diff
@@ -21,6 +21,8 @@ public SecuritySettings(IConfigurationSource source)
             var extraHeaders = source?.GetString(ConfigurationKeys.AppSecExtraHeaders);
             ExtraHeaders = !string.IsNullOrEmpty(extraHeaders) ? extraHeaders.Split(',') : Array.Empty<string>();
             KeepTraces = source?.GetBool(ConfigurationKeys.AppSecKeepTraces) ?? true;
+            var traceRateLimit = (source?.GetInt32(ConfigurationKeys.AppSecTraceRateLimit)).GetValueOrDefault(0);
```

Comment:
not PR-blocking, just idiomatic syntax ("if this is null then return 0")
```suggestion
            var traceRateLimit = source?.GetInt32(ConfigurationKeys.AppSecTraceRateLimit) ?? 0;
```